install/kubernetes/cilium/charts/agent/templates/clusterrole.yaml

```diff
@@ -55,11 +55,13 @@ rules:
   resources:
   - customresourcedefinitions
   verbs:
+  # Deprecated for removal in v1.10
   - create
-  - get
   - list
   - watch
   - update
+
```

Comment:
Please add a comment here stating that the "why" we need this get, which we only need it for the preflight so ideally we should really remove it from the agent.